web/src/app/environments/releases/release.service.ts

```diff
@@ -11,4 +11,10 @@ export class ReleaseService {
   findAllByEnvironmentId(environmentId: string): Observable<List<Release>> {
     return this.http.get<List<Release>>(`{api}/environments/${environmentId}/releases`);
   }
+
+  add(environmentId: string, release: Release, state: string): Observable<Release> {
+    return this.http.post<Release>(`{api}/environments/${environmentId}/deployed/${release.token.id}/${state}`, {
+      release: release.version
```

Comment:
Missing ending semi colon

Reply:
I am fixing the API bug you found yesterday, and I will add it to this PR, I will add the missing semi colon above 🤓

Reply:
Remind me to export my IntelliJ settings, so the IDE highlights the lint errors